### PR TITLE
Give the possibility to disable a task

### PR DESCRIPTION
**Pull request by achretien**
We can now disable a task from the config file.

Add `enabled` true by default in base task class

### CODE REVIEW

File: configs/config.json.example [hunk 1]
```diff
@@ -7,23 +7,35 @@
     "libencrypt_location": "",
     "tasks": [
       {
-        "type": "HandleSoftBan"
+        "type": "HandleSoftBan",
+        "config": {
```
Comment (TheSavior) [inline configs/config.json.example]: I don't think we should add these to the examples. It is clutter and people will think it is required. Maybe add some other task and say enabled false, just to give an example.

File: pokemongo_bot/tree_config_builder.py [hunk 2]
```diff
@@ -61,7 +61,8 @@ def build(self):
                 )
 
             instance = worker(self.bot, task_config)
-            workers.append(instance)
+            if instance.enabled:
```
Comment (TheSavior) [inline pokemongo_bot/tree_config_builder.py]: Please add a unittest for this case.

Comment (achretien) [inline pokemongo_bot/tree_config_builder.py]: done!

File: configs/config.json.example (hunk 1 above)
Comment (achretien) [inline configs/config.json.example]: I added the NicknamePokemon task with enabled:false and removed the config from other tasks

**Comment by TheSavior** [comment]
This is great, thanks!

:+1:

**Comment by vicch** [comment]
With this feature all `.json.*.example` can be combined into one.

**Pull request merged**